docs/features/event-handler/bedrock-agents.md

```diff
@@ -0,0 +1,142 @@
+
+When passing the tool parameters to your handler, we will automatically cast them to the appropriate type based on the `type` field defined in the action group. This means you can use native JavaScript types like `string`, `number`, `boolean` without worrying about parsing them yourself.
+
+Currently, we don't support parsing `array` types, so you will receive them as strings.
+
+=== "Define a tool"
+
+  ```typescript hl_lines="4 6 20"
+  --8<-- "examples/snippets/event-handler/bedrock-agents/gettingStartedFunctionsTool.ts"
+  ```
+  
+  1. The `description` field is optional, but highly recommended in the action group definition.
+
+## Advanced
+
+### Handling errors
+
+By default, we will handle errors gracefully and return a well-formed response to the agent so that it can continue the conversation with the user.
+
+When an error occurs, we send back an error message in the response body that includes the error type and message. The agent will then use this information to let the user know that something went wrong.
+
+If you want to handle errors differently, you can return a `BedrockFunctionResponse` with a custom `body` and `responseState` set to `FAILURE`. This is useful when you want to abort the conversation.
+
+!!! tip
+    You can use the same technique to reprompt the user for missing information or for them to correct their input. Just return a `BedrockFunctionResponse` with a custom message and `responseState` set to `REPROMPT`.
+
+=== "Custom error handling"
+
+    ```typescript hl_lines="19-25"
+    --8<-- "examples/snippets/event-handler/bedrock-agents/stopConversation.ts"
+    ```
+
+### Accessing Lambda context and event
+
+You can access to the original Lambda event or context for additional information. These are passed to the handler function as optional arguments.
+
+=== "Access event and context"
+
+    ```typescript hl_lines="7-9"
+    --8<-- "examples/snippets/event-handler/bedrock-agents/accessEventAndContext.ts"
+    ```
+
+### Setting session attributes
+
+When Bedrock Agents invoke your Lambda function, it can pass session attributes that you can use to store information across multiple interactions with the user. You can access these attributes in your handler function and modify them as needed.
+
+=== "Working with session attributes"
+
+    ```typescript hl_lines="24-27"
+    --8<-- "examples/snippets/event-handler/bedrock-agents/sessionAttributes.ts"
+    ```
+
+### Logging
+
+By default, the `BedrockAgentFunctionResolver` uses the global `console` logger and emits only warnings and errors.
+
+You can change this behavior by passing a custom logger instance to the `BedrockAgentFunctionResolver` constructor and setting its log level. Alternatively, you can also enable [Lambda Advanced Logging Controls](https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs-advanced.html) and setting the log level to `DEBUG`.
```

Comment:
Why not add the Powertools log level explanation here instead of ALC? Setting the Powertools log level will solve the problem both in local environment (testing, mock API and other things) and in production, running on Lambda. ALC will solve it only in production.

I would not mention ALC in this doc.

Reply:
Thanks for detailed explanation @dreamorosi! Keep it!!